src/mlpack/tests/augmented_rnns_tasks_test.cpp

```diff
@@ -54,8 +54,11 @@ class HardCodedCopyModel
       size_t& addVar = (input.at(i, 0) == 0) ? zeroCnt : oneCnt;
       ++addVar;
     }
-    assert(oneCnt % zeroCnt == 0);
-    nRepeats = oneCnt / zeroCnt;
+
+    if (zeroCnt != 0) {
```

Comment:
Hi @gaurav1086, could you move the opening bracket of the if condition on the next line.

Reply:
Also failure of assert statement causes runtime error, if statement will not do so , I think rather than an if condition adding a condition in assert statement or another assert statement might be a better idea (assert will print  a message quoting the assert condition if it fails) which might help in debugging in case it is needed.
However a member of this organization might be able to give you better insights.
Thanks.

Reply:
Do we have any reason to believe that divide by zeros actually happen here in practice?

Reply:
@kartikdutt18 , thanks for the review. I changed it another assert instead.

Reply:
@rcurtin not a practical use case but if the size of the input is zero (input.n_rows = 0).